llms.txt

```diff
@@ -0,0 +1,280 @@
+
+### 5. Handle Errors Gracefully
+Common error scenarios and solutions:
+- **No nearby stations**: Increase search radius
+- **Station not found**: Use name search first
+- **Invalid coordinates**: Verify Paris area bounds
+- **Rate limited**: Wait for reset time
+
+## Common Usage Patterns
+
+### Journey Planning Workflow
+1. `find_nearby_stations` → Find pickup options
+2. `find_nearby_stations` → Find dropoff options
+3. `plan_bike_journey` → Get optimal recommendations
+
+### Station Discovery Workflow
+1. `search_stations_by_name` → Find by landmark
+2. `get_station_by_code` → Get detailed info
+3. `find_nearby_stations` → Find alternatives
+
+### Area Analysis Workflow
+1. `get_area_statistics` → Overview of region
+2. `find_nearby_stations` → Detailed station list
+3. `get_station_by_code` → Specific station details
+
+## Error Handling
+
+### Common Error Codes
+- `-32001`: Station not found
+- `-32002`: Invalid coordinates
+- `-32003`: Outside service area
+- `-32004`: Search radius too large
+- `-32005`: Result limit exceeded
+
+### Recovery Strategies
+- **Expand search criteria**: Increase radius or limits
+- **Validate inputs**: Check coordinate bounds
+- **Try alternatives**: Use name search vs. code lookup
+- **Check service status**: Use health endpoint
+
+## Performance Considerations
+
+### Response Times
+- Station lookup: 30-100ms
+- Nearby search: 50-200ms
+- Area statistics: 100-500ms
+- Journey planning: 200-800ms
+
+### Rate Limits
+- Tools: 100 requests/minute
+- Resources: 60 requests/minute
+- Burst: 10 requests/second
+
+### Caching Strategy
+- Cache reference data locally when possible
+- Respect TTL values for real-time data
+- Use health endpoint to verify data freshness
+
+## Integration Examples
+
+### Basic Station Search
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "tools/call",
+  "params": {
+    "name": "find_nearby_stations",
+    "arguments": {
+      "latitude": 48.8566,
+      "longitude": 2.3522,
+      "radius_meters": 500,
+      "limit": 5
+    }
+  },
+  "id": 1
+}
+```
+
+### Journey Planning
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "tools/call",
+  "params": {
+    "name": "plan_bike_journey",
+    "arguments": {
+      "origin": {"latitude": 48.8606, "longitude": 2.3376},
+      "destination": {"latitude": 48.8584, "longitude": 2.2945},
+      "preferences": {
+        "bike_type": "electric",
+        "max_walk_distance": 400
+      }
+    }
+  },
+  "id": 2
+}
+```
+
+### Get Documentation
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "docs/schema",
+  "params": {
+    "format": "json",
+    "include_examples": true,
+    "token_efficient": false
+  },
+  "id": 3
+}
+```
+
+## Support and Troubleshooting
+
+### Health Monitoring
+Use the health endpoint to check system status:
+```
+GET /resources/velib://health
+```
+
+### Documentation Updates
+The schema documentation is generated dynamically and always reflects the current API state. Access it via:
```

Comment:
"dynamically" and "hardcoded" have very different meanings. 

Please don't advertise the docs as being "dynamically generated;" I think it's misleading.